### PR TITLE
[KIECLOUD-115] cekit for rhel builds

### PR DESCRIPTION
**Pull request by tchughesiv**
- pushes to dist-git and kicks off build in OSBS

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+schema_version: 1
+
+name: "rhpam-7-tech-preview/business-automation-operator"
+description: "Red Hat Business Automation Operator"
+version: "x.x"
```
Comment (ruromero) [inline image.yaml]: Can't you just remove this line? Is it perhaps required?

Comment (tchughesiv) [reply image.yaml]: it's required... just verified.

**Comment by tchughesiv** [comment]
added note in README

**Pull request merged**